test/test_ccf.py

```diff
@@ -141,3 +141,39 @@ def submit(**kwargs):
         submit(alg="ES256", kty="ec", ec_curve="P-256")
         submit(alg="ES384", kty="ec", ec_curve="P-384")
         submit(alg="PS256", kty="rsa")
+
+
+def test_accepted_did_issuers(tmp_path: Path):
+    def not_allowed(f):
+        with pytest.raises(ServiceError, match="InvalidInput: Unsupported did issuer in protected header"):
+            f()
+
+    with SCITTFixture(tmp_path) as fixture:
+
+        def submit(**kwargs):
```

Comment:
For this test, IMO it is fine to use a single identity and signed claims throughout the test so no need for a helper function. Just define an `identity` and `claims` at the top and use that everywhere by calling `fixture.client.submit_claim` directly.